crates/autopilot/src/driver_model.rs

```diff
@@ -136,6 +137,16 @@ pub mod solve {
         pub call_data: Vec<u8>,
     }
 
+    #[serde_as]
+    #[derive(Clone, Debug, Default, Deserialize)]
+    #[serde(rename_all = "camelCase", deny_unknown_fields)]
+    pub struct OrderAmounts {
+        #[serde_as(as = "HexOrDecimalU256")]
+        pub in_amount: U256,
+        #[serde_as(as = "HexOrDecimalU256")]
+        pub out_amount: U256,
```

Comment:
Comments as to whether these amount are supposed to take fees into account would be helpful.